cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-analytics.js

```diff
@@ -582,19 +583,18 @@ const handleSurveyResultsSavePdfClick = (event) => {
  * @returns {object} Event data.
  */
 const handleSurveyResultsGetLinkClick = (event) => {
-  const link = closest(
-    event.target,
+  const link = event.target.closest(
     '#modal-share-url .tdp-survey__initials-set'
   );
 
   if (!link || !link.classList.contains('a-btn')) {
     return;
   }
-  const textField = queryOne(
+  const textField = document.querySelector(
     '#modal-share-url input#modal-share-url-initials-input'
```

Comment:
For another PR, I wonder if this could be `input#modal-share-url-initials-input` and doesn't need to the leading ID, since the input is an ID.